app/http.php

```diff
@@ -58,6 +58,18 @@
 $register->set('logger', function () {
     $providerName = Http::getEnv('OPR_EXECUTOR_LOGGING_PROVIDER', '');
     $providerConfig = Http::getEnv('OPR_EXECUTOR_LOGGING_CONFIG', '');
+
+    try {
+        $loggingProvider = new DSN($providerConfig ?? '');
+
+        $providerName = $loggingProvider->getScheme();
+        $providerConfig = match ($providerName) {
+            'sentry' => ($loggingProvider->getUser() ?? '') . ';' . $loggingProvider->getHost(),
+            default => $loggingProvider->getHost(),
+        };
+    } catch (Throwable) {
+    }
```

Comment:
Same comments as the Appwrite PR. We need to adapt this code block to the new constructor syntax of the logging library.